### PR TITLE
Move api-docs from appcenter-docs to specific repository

### PR DESCRIPTION
**Pull request by cinlloc**
- Remove intersphinx mapping for API doc, use current API doc homepage instead

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,13 +35,11 @@ Launch the [venv.sh](blob/master/venv.sh) script to prepare the virtualenv or to
```
Comment (segalaj) [inline README.md]: Please update l.28:
`You can also build each guide individually, note though the end user guide and admin guide depend on the API guide.`

File: changelog.rst [hunk 2]
```diff
@@ -1067,3 +1067,5 @@ Bug Fixes
 * 5551 Spelling mistake in UI when publishing to Flexiant
 * 5549 The error of Keystone version is displayed in Keystone Server URL
 * 5403 Scan fails when trying to rebuild a non repo package
+
+.. _apidoc: http://docs.usharesoft.com/projects/appcenter-api-guide/en/latest/
```
Comment (segalaj) [inline changelog.rst]: shouldn't it be the new api doc project here?

Comment (cinlloc) [reply changelog.rst]: > I don't understand if you want to go like this, or if it need to wait the new API doc project?

We have to wait new API doc project before removing it from here

Comment (cinlloc) [reply changelog.rst]: Yes. I assumed new API doc address was the same as old one, but if not we'll have to change it here.

File: README.md (hunk 1 above)
Comment (cinlloc) [reply README.md]: Nice catch, I'll fix it!

**Comment by lqueiroga** [comment]
Thank for this PR

**Pull request merged**